Applications/Spire/Source/KeyBindings/KeyBindingsWindow.cpp

```diff
@@ -0,0 +1,18 @@
+#include "Spire/KeyBindings/KeyBindingsWindow.hpp"
+
+using namespace boost::signals2;
+using namespace Spire;
+
+KeyBindingsWindow::KeyBindingsWindow(const KeyBindings& key_bindings,
+  QWidget* parent)
+  : Window(parent),
+    m_key_bindings(key_bindings) {}
+
+const KeyBindings& KeyBindingsWindow::get_key_bindings() const {
+  return m_key_bindings;
```

Comment:
Not that important really, but if you want a pure stub without touching the header file, you can always create a static instance and return a reference to it.
Feel free to keep at as is.